content/guides/hosting/hosting-options/self-managed/kubernetes-operator/operator-airgapped.md

```diff
@@ -122,7 +122,7 @@ Along with the container images, you also must ensure that the following Helm ch
 - [W&B Platform](https://github.com/wandb/helm-charts/tree/main/charts/operator-wandb)
 
 
-The `operator` chart is used to deploy the W&B Operator, or the Controller Manager. While the `platform` chart is used to deploy the W&B Platform using the values configured in the custom resource definition (CRD).
+The `operator` chart is used to deploy the W&B Operator aka the Controller Manager. While the `platform` chart is used to deploy the W&B Platform using the values configured in the custom resource definition (CRD).
 
```

Comment:
`aka` is subject to review :)

Reply:
Are we waiting on anyone else to weigh in?